packages/search/addon/components/navi-search-result/report.js

```diff
@@ -0,0 +1,17 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class Report extends Component {
+  @service('navi-report-search-provider') reportSearchProvider;
+
+  @tracked searchResult;
+
+  title = 'Reports & Dashboards';
+
+  @action
+  async search(element, query) {
+    this.searchResult = await this.reportSearchProvider.search(query);
```

Comment:
should we either have a javadoc explaining that this sets the search result property, or define the searchResult property as a property of the component?